modules/members.js

```diff
@@ -0,0 +1,101 @@
+// @ts-check
+
+const Augur = require("augurbot-ts"),
+  u = require("../utils/utils"),
+  c = require("../utils/modCommon"),
+  Discord = require("discord.js"),
+  config = require("../config/config.json");
+
+/**
+ * Creates a profile card - a PNG that contains some user information in a fun format!
+ * @param {Discord.GuildMember} member The member to create the profile card for.
+ * @returns {Promise<Buffer|undefined>} File-like object to attach to your response.
+ */
+async function makeProfileCard(member) {
+  try {
+    const badgeData = require("../utils/badges"),
+      RankInfo = require("../utils/RankInfo"),
+      Jimp = require("jimp");
+
+    const badgePath = "./media/badges/";
+    const font = await Jimp.loadFont(Jimp.FONT_SANS_16_WHITE);
+    const card = await Jimp.read("./media/background.jpg");
```

Comment:
Same thing as above, and you can do `card.clone()` to edit a copy instead of the original.

Reply:
Implemented